zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromProcessInstanceHandler.java

```diff
@@ -130,24 +130,7 @@ public void updateEntity(
       piEntity
           .setParentProcessInstanceKey(recordValue.getParentProcessInstanceKey())
           .setParentFlowNodeInstanceKey(recordValue.getParentElementInstanceKey());
-      //      if (piEntity.getTreePath() == null) {
-      //        final String treePath = getTreePathForCalledProcess(recordValue);
-      //        piEntity.setTreePath(treePath);
-      //      }
     }
-
-    // TODO: Implement properly treePath https://github.com/camunda/camunda/issues/18378
-    final String treePath = getTreePathForCalledProcess(recordValue);
-    piEntity.setTreePath(treePath);
-    //
-    //    if (piEntity.getTreePath() == null) {
-    //      final String treePath =
-    //          new TreePath()
-    //
-    // .startTreePath(ConversionUtils.toStringOrNull(recordValue.getProcessInstanceKey()))
-    //              .toString();
-    //      piEntity.setTreePath(treePath);
-    //    }
```

Comment:
@sdorokhova what I thought we could do is to always set a basic treePath, when we are on a flat PI

Like we have PI and flow scope and key. I guess we could at least set for the ones where flow scope is either -1 or same as PI key? 🤔

Reply:
@Zelldon I decided to set default tree path value for ELEMENT_ACTIVATING and BpmnElementType=PROCESS as this is the first record that comes for process instance before any incident record